packages/playwright-test/src/cli.ts

```diff
@@ -107,13 +107,18 @@ Examples:
 async function runTests(args: string[], opts: { [key: string]: any }) {
   await startProfiling();
 
+  const os = require('os');
+  const cpus = os.cpus().length;
+  const isLikelyM1 = os.arch() === 'arm64' && os.platform() === 'darwin';
```

Comment:
We have a version that is not fooled by Rosetta: https://github.com/microsoft/playwright/blob/10bf5f3e493d25a72eda0f530590bafbab46edc5/packages/playwright-core/src/utils/utils.ts#L515-L535